File: 0088-merge-sorted-array/0088-merge-sorted-array.java
import java.util.Arrays.*;
class Solution {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
        int i,j;
        j=0;
        for(i=m;i<m+n;i++){
            nums1[i]=nums2[j];
            j++;
        }
        for(i = m; i < nums1.length; i++) {
			int temp = nums1[i];
			int k = i - 1;
			
			while(k >= 0 && nums1[k] > temp) {
				nums1[k+1] = nums1[k];
				k--;
			}
			nums1[k+1] = temp;
		}
        //Arrays.sort(nums1);
    }
}

File: 0088-merge-sorted-array/0088-merge-sorted-array.py
class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        """
        Do not return anything, modify nums1 in-place instead.
        """
        a=n-1;
        b=m-1;
        i=m+n-1
        while a>=0:
            if (b>=0 and nums1[b]>nums2[a]):
                nums1[i]=nums1[b];
                i=i-1
                b=b-1
            else:
                nums1[i]=nums2[a]
                i=i-1
                a=a-1
        return nums1